pkg/otlp/internal/serializerexporter/consumer.go

```diff
@@ -124,5 +142,25 @@ func (c *serializerConsumer) flush(s serializer.MetricSerializer) error {
 			sketchesErr = s.SendSketch(sketchesSource)
 		})
 
+	c.sendAPMStats()
 	return multierr.Append(serieErr, sketchesErr)
 }
+
+func (c *serializerConsumer) sendAPMStats() {
+	addr := fmt.Sprintf("http://localhost:%s/v0.6/stats", config.Datadog.GetString("apm_config.receiver_port"))
```

Comment:
The datadogprocessor will compute stats of all the traces that pass through it. If those traces don't reach our intake because of configuration or (fully exclusive) sampling, then yes, there could be (valid) stats without traces.

As for traces without stats. I don't think that can happen as long as the processor is used. If the processor is not used, then we are where we are today: stats only for sampled traces.

Reply:
I was thinking about the failure mode where sending stats to `/v0.6/stats` would fail but sending traces to the `5003` port would not because something is set up incorrectly in the configuration (say some configuration flag that only affects one or the other). This seems pretty straightforward so probably there is no such case, but wanted to double check with you.

Reply:
I'm not aware of any such case. The receiver port needs to be correct and active for traces to come in. I don't think that's ever disabled (except when running the agent in the collector exporter, but that's not the case here since we are in the Datadog Agent and the trace-agent receiver is on by default).

As for the 5003 internal port, I don't see why users would want to change that, unless it clashes with something. But then again, if traces don't work, it's usually a higher priority than stats so it should be noticed quicker.